arch/x86/pagetables.c

```diff
@@ -216,10 +216,10 @@ static inline void pgentry_fixup_flags(pgentry_t *entry, unsigned long flags) {
     if (unlikely(*entry != entry_new)) {
         char flags_str_old[16];
         char flags_str_new[16];
-        printk("WARNING: Already-present PTE protection flags conflicts with our.\n"
-               "         Updating present flags: %s -> %s\n",
-               dump_pte_flags(flags_str_old, 16, (pte_t) *entry),
-               dump_pte_flags(flags_str_new, 16, (pte_t) entry_new));
+        warning("Already-present PTE protection flags conflicts with our.\n"
```

Comment:
s/conflicts/conflict/
Hehe sorry I think that was me who originally worte hehe

Reply:
Fixed. Thanks.